### DISCUSSION TITLE
Welcome to AAChartKit Discussions!

### DISCUSSION THREAD

**Discussion by AAChartModel** [AAChartModel/AAChartKit#1103] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by majharuli0** [comment]
After Updaing my iOS SDK version to  18 & Xcode 16.2 this issue happening do you have any idea why:
ld: library 'AAChartKit' not found
clang: error: linker command failed with exit code 1 (use -v to see invocation)

(Swift)
